tests/rest/client/test_relations.py

```diff
@@ -311,14 +313,17 @@ def test_pagination_from_sync_messages(self):
         self.assertEquals(200, channel.code, channel.json_body)
         messages_end = channel.json_body["end"]
         self.assertIsNotNone(messages_end)
-        # Ensure the event above is not in the chunk.
+        # Ensure the relation event is not in the chunk returned from /messages.
         self.assertNotIn(
             annotation_id, [ev["event_id"] for ev in channel.json_body["chunk"]]
         )
 
-        # Request the relations with the token from sync/messages -- this is a
-        # tiny bit silly since how do we know the parent ID? Just assume we had
-        # it from an earlier sync and the above are gappy.
+        # Request /relations with the pagination token from the /sync and /messages
+        # requests above.
```

Comment:
This sounds like it suggests that the pagination token from `/sync` and `/messages` is identical.
If that's true, I would perhaps like it to be clarified (since the for loop makes the opposite suggestion). If it's not, I would tweak a bit
```suggestion
        # Request /relations with the pagination tokens from both the /sync and /messages
        # requests above, in turn.
```

Reply:
I think they actually are (just due to the nature of the data in the unit test), but that's not important to the test.